### PR TITLE
Upgrade to camel-3.19.0

### PR DESCRIPTION
**Pull request by JiriOndrusek**
Switch from camel-3.19.0-SNAPSHOT to camel-3.19.0

### CODE REVIEW

**Comment by JiriOndrusek** [comment]
@zbendhiba Yes this is the PR. I changed to title to make it clear.

**Comment by JiriOndrusek** [comment]
@jamesnetherton

- `antora.yml` is temporarily reverted to Camel 3.18.x to not break website. 
- I switched from draft to ready for merge
- I think that this PR could be merged and once the doc for 3.19 is available we can update `antora.yml`

-

**Review by jamesnetherton — APPROVED** [review]

**Review by zbendhiba — APPROVED** [review]

**Comment by jamesnetherton** [comment]
Workaround for the example project failure is here https://github.com/apache/camel-quarkus-examples/pull/114.

File: integration-tests/quartz/src/main/resources/application.properties [hunk 1]
```diff
@@ -18,3 +18,6 @@
 camel.component.quartzFromProperties.propertiesFile = quartz.properties
 
 quarkus.native.resources.includes = quartz.properties
+
+#quick workaround caused by https://issues.apache.org/jira/browse/CAMEL-18143
```
Comment (zbendhiba) [inline integration-tests/quartz/src/main/resources/application.properties:22]: Is this an isssue ?

Comment (jamesnetherton) [reply integration-tests/quartz/src/main/resources/application.properties:22]: There is a PR to fix that. But it depends on something that'll be available in Quarkus 2.14:

https://github.com/apache/camel-quarkus/pull/4153

Comment (zbendhiba) [reply integration-tests/quartz/src/main/resources/application.properties:22]: Many thanks! I wanted to make sure there's an open issue

**Pull request merged**